frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html

```diff
@@ -1,12 +1,4 @@
-<div *ngIf="spinnerService.isLoading$ | async">
-    <div
-        [ngStyle]="{
-            'font-size': size,
-            margin: margin,
-            top: top,
-            left: left,
-            position: position
-        }"
-        class="loader"
-    ></div>
+<div *ngIf="spinner.isLoading$ | async">
+    <div class="loader"></div>
+    <p class="loader-text">Loading...</p>
```

Comment:
we don't need this is the spinner component

Reply:
you mean we don’t need this text at all or move it to another place?

Reply:
according to the task- "Loader should  be interactive"

Reply:
removed